internal/ioutil/io_test.go

```diff
@@ -50,3 +54,62 @@ func TestUnwrapNopCloser(t *testing.T) {
 		})
 	}
 }
+
+func TestCopyBuffer(t *testing.T) {
+	type args struct {
+		src  io.Reader
+		buf  []byte
+		desc ocispec.Descriptor
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantDst string
+		wantErr error
+	}{
+		{
+			name:    "exact buffer size, no errors",
+			args:    args{bytes.NewReader([]byte("foo")), make([]byte, 3), content.NewDescriptorFromBytes("test", []byte("foo"))},
```

Comment:
nit: should we make `[]byte("foo")` a variable?

Reply:
Added this variable.